api/src/LinkyLink/Controllers/ValidatePageController.cs

```diff
@@ -1,73 +1,69 @@
-using System;
+﻿using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.Azure.WebJobs;
-using Microsoft.Azure.WebJobs.Extensions.Http;
+using HtmlAgilityPack;
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
-using OpenGraphNet;
-using System.Linq;
 using Newtonsoft.Json.Linq;
-using System.Collections.Generic;
-using HtmlAgilityPack;
+using OpenGraphNet;
 using LinkyLink.Models;
 
-namespace LinkyLink
+namespace LinkyLink.Controllers
 {
-    public partial class LinkOperations
+    [Route("api/[controller]")]
+    [ApiController]
+    // Todo - Change the name of the controller
+    public class ValidatePageController : Controller
     {
-        [FunctionName(nameof(ValidatePage))]
-        public async Task<IActionResult> ValidatePage(
-            [HttpTrigger(AuthorizationLevel.Function, "POST", Route = "validatePage")] HttpRequest req,
-            ILogger log)
```

Comment:
per the TODO below it seems that the plan is to add logging later, is that correct? if so plz ignore my comment, otherwise, can we add logging for now?

Reply:
Logging will be implemented later. I want to understand App Insights/Azure Monitor and then implemented logging and don`t want to hold this PR.